sass/main.scss

```diff
@@ -680,121 +680,304 @@ nav {
+        cursor: pointer;
+        color: white;
+    }
+
+    .day {
+        .number {
+            font-size: 6.2em;
+            font-weight: $font-bold;
+            line-height: 0.9; // TODO: Can this be done cleaner?
+        }
+
+        .name {
+            font-size: 1.6em;
+            font-weight: $font-regular;
+
+        }
+
+        &:not(:last-child) {
+            .name {
+                margin-bottom: 10px;
+            }
+        }
+    }
+
+    .events {
+        width: 60%;
+
+        // Sequentially assign colors to all elements
+        .event:nth-child(3n) {
+            border-top: 10px solid $red;
+        }
+        .event:nth-child(3n + 1) {
+            border-top: 10px solid $orange;
+        }
+        .event:nth-child(3n + 2) {
+            border-top: 10px solid $light-gray;
+        }
+
+        .event {
+            background: white;
+            border-radius: 15px;
+            margin: 10px 0px;
+            padding: 10px;
+            position: relative;
+
+            &.past {
+                background-color: #bfbfbf;
+            }
+
+            .time {
+                color: $light-blue;
+            }
+
+            .title {
+                color: black;
+                font-size: 1.3em;
+            }
+
+            .marker {
+                $marker-width: 20px;
 
-                    border-bottom: 2px solid $red;
-                    border-top: 2px solid $red;
+                border-bottom: 2px solid $red;
+                border-top: 2px solid $red;
+                position: absolute;
+                left: -25px;
+                right: 0;
+                top: 0;
+
+                &::before {
+                    content: "";
+                    background-color: $red;
+                    border-radius: 50%;
                     position: absolute;
-                    left: -25px;
-                    right: 0;
-                    top: 0;
-
-                    &::before {
-                        content: "";
-                        background-color: $red;
-                        border-radius: 50%;
-                        position: absolute;
-                        bottom: -($marker-width / 2);
-                        left: -($marker-width / 2);
-                        height: $marker-width;
-                        width: $marker-width;
-                    }
+                    bottom: -($marker-width / 2);
+                    left: -($marker-width / 2);
+                    height: $marker-width;
+                    width: $marker-width;
                 }
             }
         }
+    }
+
+    // Note that pre-event style overrides default schedule content styling
+    #pre-event-content {
+        display: flex;
+        flex-direction: column;
+        flex-wrap: none;
+
+        .day-row {
+            display: flex;
+            justify-content: space-between;
+        }
+
+        // Coordinate color with day on left and event border on top
+        .pre-16 {
```

Comment:
Why is this a class? I only see it in the html once

Reply:
my brain does one thing and one thing well: make CSS classes

Reply:
#1161